dubbo-compatible/src/main/java/com/alibaba/dubbo/rpc/RpcResult.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.dubbo.rpc;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RpcResult extends AppResponse implements com.alibaba.dubbo.rpc.Result {
+    public RpcResult() {
+    }
+
+    public RpcResult(Object result) {
+        super(result);
+    }
+
+    public RpcResult(Throwable exception) {
+        super(exception);
+    }
+    
+    @Override
+    public org.apache.dubbo.rpc.Result whenCompleteWithContext(BiConsumer<org.apache.dubbo.rpc.Result, Throwable> fn) {
+        return null;
+    }
```

Comment:
What is the purpose of adding this?

Reply:
Wait a moment, I found when dubbo3 using dubbo2's Filter, RpcResult will be handle as AsyncRpcResult. 
This function whenCompleteWithContext will be changed later, more code need to fix.